corehq/apps/domain/_design/views/docs/map.js

```diff
@@ -1,14 +0,0 @@
-function (doc) {
-    // !code util/doc_utils.js
-
-    var i,
-        date,
-        domains;
-    if (doc.doc_type) {
-        date = get_date(doc);
-        domains = get_domains(doc);
-        for (i = 0; i < domains.length; i += 1) {
-            emit([domains[i], doc.doc_type, date], null);
```

Comment:
Hey @esoergel, it looks like [`copy_domain`](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/domainsync/management/commands/copy_domain.py#L173) uses this view heavily. Isn't this the best method for copy_domain? Should we get this back?